### ISSUE TITLE
SRTM does not work with filmic-blender enabled

### ISSUE THREAD

**Issue by robsl2314** [domlysz/BlenderGIS#268]
**Using: Blender 2.80 in Windows 10**

The SRTM functionality does not work if [filmic-blender](https://sobotka.github.io/filmic-blender/) is installed.

## Stacktrace:
```
WARNING:BlenderGIS-master.core.georaster.georaster:145:Cannot extract georefencing informations from tif tags
Traceback (most recent call last):
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\io_import_georaster.py", line 377, in execute
    grid = bpyGeoRaster(filePath, subBoxGeo=subBox, clip=self.clip, fillNodata=self.fillNodata, useGDAL=HAS_GDAL, raw=True)
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\utils\georaster_utils.py", line 259, in __init__
    self._load()
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\utils\georaster_utils.py", line 275, in _load
    self.bpyImg.colorspace_settings.name = 'Non-Color'
TypeError: bpy_struct: item.attr = val: enum "Non-Color" not found in ('AppleP3 Filmic Log Encoding', 'AppleP3 sRGB OETF', 'BT.1886 EOTF', 'BT.1886 Filmic Log Encoding', 'Filmic Log Encoding', 'Linear', 'Non-Colour Data', 'sRGB OETF')

location: C:\Program Files\Blender Foundation\Blender\2.80\scripts\modules\bpy\ops.py:199

location: C:\Program Files\Blender Foundation\Blender\2.80\scripts\modules\bpy\ops.py:199
Error: Traceback (most recent call last):
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\io_import_georaster.py", line 377, in execute
    grid = bpyGeoRaster(filePath, subBoxGeo=subBox, clip=self.clip, fillNodata=self.fillNodata, useGDAL=HAS_GDAL, raw=True)
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\utils\georaster_utils.py", line 259, in __init__
    self._load()
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\utils\georaster_utils.py", line 275, in _load
    self.bpyImg.colorspace_settings.name = 'Non-Color'
TypeError: bpy_struct: item.attr = val: enum "Non-Color" not found in ('AppleP3 Filmic Log Encoding', 'AppleP3 sRGB OETF', 'BT.1886 EOTF', 'BT.1886 Filmic Log Encoding', 'Filmic Log Encoding', 'Linear', 'Non-Colour Data', 'sRGB OETF')

location: C:\Program Files\Blender FoundatiTraceback (most recent call last):
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\io_get_srtm.py", line 43, in invoke
    return self.execute(context)#context.window_manager.invoke_props_dialog(self)
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\io_get_srtm.py", line 135, in execute
    fillNodata = False)
  File "C:\Program Files\Blender Foundation\Blender\2.80\scripts\modules\bpy\ops.py", line 199, in __call__
    ret = op_call(self.idname_py(), C_dict, kw, C_exec, C_undo)
RuntimeError: Error: Traceback (most recent call last):
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\io_import_georaster.py", line 377, in execute
    grid = bpyGeoRaster(filePath, subBoxGeo=subBox, clip=self.clip, fillNodata=self.fillNodata, useGDAL=HAS_GDAL, raw=True)
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\utils\georaster_utils.py", line 259, in __init__
    self._load()
  File "%user%\Roaming\Blender Foundation\Blender\2.80\scripts\addons\BlenderGIS-master\operators\utils\georaster_utils.py", line 275, in _load
    self.bpyImg.colorspace_settings.name = 'Non-Color'
TypeError: bpy_struct: item.attr = val: enum "Non-Color" not found in ('AppleP3 Filmic Log Encoding', 'AppleP3 sRGB OETF', 'BT.1886 EOTF', 'BT.1886 Filmic Log Encoding', 'Filmic Log Encoding', 'Linear', 'Non-Colour Data', 'sRGB OETF')

location: C:\Program Files\Blender Foundation\Blender\2.80\scripts\modules\bpy\ops.py:199
```

## How to Reproduce:
- Install filmic-blender
- Start Blender
- Insert Basemap via GIS Plugin
- Call "Get SRTM" via GIS Plugin
- >> Stacktrace

**Comment by domlysz** [comment]
don't know what to say here as the code respect the Blender Python API : 
https://docs.blender.org/api/current/bpy.types.ColorManagedInputColorspaceSettings.html

The color space of an image is a value in 'NONE', ‘Filmic Log’, ‘Linear’, ‘Linear ACES’, ‘Non-Color’, ‘Raw’, ‘sRGB’, ‘XYZ’. Here the script set the color space to 'Non-Color' but seems this value does not exists in filmic-blender but it works as expected with standard Blender.

**Comment by domlysz** [comment]
so filmic-blender break the python api, no developer wants to handle this kind of situation

**Comment by s-leger** [comment]
You should report to filmic-blender addon developper.
In the meantime as workaround you may check for 'Raw' and  'Non-Colour Data'
`
        try:
            # fix issue #137 support for Filmic Blender addon
            image.colorspace_settings.name = 'Non-Colour Data'
        except:
            pass
        try:
            image.colorspace_settings.name = 'Raw'
        except:
            pass
`

**Comment by domlysz** [comment]
should be fixed

https://github.com/domlysz/BlenderGIS/commit/f3c7ba4172600d296366f62e9fb73a255ff603aa
